src/typescript-service.ts

```diff
@@ -51,37 +52,37 @@ export class TypeScriptService implements LanguageHandler {
 		this.traceModuleResolution = traceModuleResolution || false;
 	}
 
+	// TODO pass remoteFs and strict to constructor
```

Comment:
I meant to not pollute the `initialize` arguments because they should be used for other stuff (cancellation token etc) and should be consistent with other method handlers